src/kili/entrypoints/mutations/asset/__init__.py

```diff
@@ -279,12 +279,12 @@ def update_properties_in_assets(
             )
             raise MissingArgumentError("Please provide either `asset_ids` or `external_ids`.")
 
-        asset_ids = get_asset_ids_or_throw_error(
-            self.kili_api_gateway, asset_ids, external_ids, project_id
+        resolved_asset_ids = UseCasesUtils(self.kili_api_gateway).get_asset_ids_or_throw_error(
+            asset_ids, external_ids, project_id  # type: ignore
```

Comment:
this will ignore type errors for all three arguments, which makes pyright useless

it's best to use typing.cast for arrays, or ProjectId(project_id) for single entities